umbra-js/src/utils/utils.ts

```diff
@@ -216,54 +225,242 @@ export async function lookupRecipient(
     supportTxHash,
   }: { advanced?: boolean; supportPubKey?: boolean; supportTxHash?: boolean } = {}
 ) {
+  const chainId = (await provider.getNetwork()).chainId;
   // Check if identifier is a public key. If so we just return that directly
   const isPublicKey = id.length === 132 && isHexString(id);
   if (supportPubKey && isPublicKey) {
     assertValidPoint(id);
-    return { spendingPublicKey: id, viewingPublicKey: id };
+    return { spendingPublicKey: id, viewingPublicKey: id, block: undefined };
```

Comment:
Is adding block here necessaary? And same for the return statements below?

Reply:
No.. thanks.. good catch: 4ba39281b8b88b651555026a3da77be4053e27d6